extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -30,7 +31,25 @@ class AmpAccordion extends AMP.BaseElement {
     /** @const @private {!NodeList} */
     this.sections_ = this.getRealChildren();
 
+    /** @const @private {string} */
+    this.id_ = this.getSessionStorageKey();
+
+    /** @const @private {!Object|undefined} */
+    this.currentState_ = null;
     this.element.setAttribute('role', 'tablist');
+
+    // sessionStorage key: special created id for this element, this.id_.
+    // sessionStorage value: string that can convert to this.currentState_ obj.
+    // TODO: add test for setting and reading sessionStorage.
+    try {
+      this.currentState_ =
+        JSON.parse(this.getWin()./*OK*/sessionStorage.getItem(this.id_));
+    } catch (e) {
+      console./*OK*/error(e.message, e.stack);
```

Comment:
I changed it to `/*REVIEW*/`, but I found it's not needed once I add the file name to whitelist. Do I still want to keep it?